submissions/AnnaStoyano/oop-exercise/index.js

```diff
@@ -0,0 +1,106 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Mammal{
+   constructor(species,name,gender,legs,friends,saying){
```

Comment:
Please install and tune up [prettier](https://prettier.io/) plugin for your IDE. It will help you (even is able to fix minor issues for you) to keep code formatting as per code style conventions (e.g. spaces between arguments in the parameters or arguments list, or before opening figure bracket `{` after class name or method/function params list, or after `if`). Tiny things but they pop out.

Just ask other students in chat how do they make their code well formatted and neat.

This is not critical **now**, but will make the code easier to read and will pay off when you will collaborate with others (at work, for example).